recipes/adept/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "Adept" %}
+{% set version = "1.1.8c24824" %}
```

Comment:
We prefer stable releases. Isn't `1.1` enough?

Reply:
@isuruf The only release is [v1.1](https://github.com/rjhogan/Adept/releases) from two years ago and ~25 commits old.  It has been fairly stable however, there were 3 commits (updating documentation) a few days ago -- but these were the first commits since April 2016.

Reply:
In particular since v1.1 were a few commits fixing the build process, and fixing an incorrect call to an integer abs function

Reply:
Can you ask upstream about a release? If it takes time for a new release, then we can package the git master.

Reply:
@isuruf I'll open an issue there / send an email

Reply:
@isuruf I haven't head anything back, would it be preferable for me to fork my own copy of the base repo and make a new release at the latest commit?